lib/index.js

```diff
@@ -24,6 +25,7 @@ module.exports = function (service) {
 module.exports.config = function (config, callback) {
   Hoek.assert(config && typeof config === 'object', 'Config must exist and be an object');
   Hoek.assert(config.backends && config.backends.length, 'Config must contain backends');
+  callback = callback || OrPromise();
```

Comment:
Thoughts on inlining some of the OrPromise functionality instead of making it a separate dependency? Since its only needed in this one spot and its a smaller function it makes more sense to keep the logic here